projects/snapshot_bot/memory.cc

```diff
@@ -1,26 +1,31 @@
 #define NO_HEADER_DEFINITIONS
 #include "memory.h"
 
-// Standard C++ includes
-#include <fstream>
-
 // BoB robotics includes
 #include "common/serialise_matrix.h"
 #include "imgproc/mask.h"
+#include "navigation/image_database.h"
 
 // BoB robotics third party includes
 #include "plog/Log.h"
 #include "third_party/path.h"
 
 // Snapshot bot includes
 #include "config.h"
+#include "image_input.h"
 
 using namespace BoBRobotics;
 using namespace units::angle;
 using namespace units::length;
 using namespace units::literals;
 using namespace units::math;
 
+constexpr const char *CSVBestHeading = "Best heading [degrees]";
```

Comment:
I'm not 100% on the rules for ``constexpr`` objects but I think these should probably be in the anonymous namespace so they're only module scope